src/main/resources/arguments.json

```
@@ -209,11 +209,16 @@
       "_type": "command"
     }
   },
-  "volume": {
+  "volume*": {
     "volume": {
       "_type": "required"
     }
   },
+  "seek*": {
+    "seek_time": {
```

```suggestion
    "timestamp": {
```
That is the name of the argument